flow/connectors/postgres/qvalue_convert.go

```diff
@@ -525,12 +524,15 @@ func numericToRat(numVal *pgtype.Numeric) (*big.Rat, error) {
 			return nil, errors.New("numeric value is infinity")
 		}
 
-		rat := new(big.Rat)
-
-		rat.SetInt(numVal.Int)
-		divisor := new(big.Rat).SetFloat64(math.Pow10(int(-numVal.Exp)))
-		rat.Quo(rat, divisor)
-
+		rat := new(big.Rat).SetInt(numVal.Int)
```

Comment:
pgtype exposes this with [AssignTo](https://pkg.go.dev/github.com/jackc/pgtype#ArrayType.AssignTo)
```go
var rat big.Rat
err := numVal.AssignTo(&rat)
if err != nil {
  return new(big.Rat), nil
}
return rat, nil
```

Reply:
We're using `github.com/jackc/pgx/v5/pgtype` which does not seem to expose this: https://github.com/jackc/pgx/blob/master/pgtype/numeric.go

Reply:
We could do something like:

```
iv := numVal.toBigInt()
var rat big.Rat
rat.SetInt(iv)
```

Reply:
https://pkg.go.dev/github.com/jackc/pgx/pgtype#Decimal.AssignTo

Reply:
Looks like they decided to prefer shopspring-decimal https://github.com/jackc/pgx/wiki/Numeric-and-decimal-support

Reply:
But we want `*big.Rat` because that's what bigquery takes, not a proper decimal type